groups/sig-architecture/groups.yaml

```diff
@@ -68,6 +68,9 @@ groups:
       - hh@ii.coop
       - jbelamaric@google.com
       - zz@ii.coop
+      - koray@kubermatic.com
+      - mario@kubermatic.com
+      - marko@kubermatic.com
```

Comment:
Can you please use `mudrinic.mare@gmail.com` as I use it for everything else in the upstream projects?

Reply:
right, maybe I should use my gmail one as well for the same reason. @mfahlandt do you have any preference?